src/popover/popover.directive.ts

```diff
@@ -33,6 +41,10 @@ export class PopoverDirective implements OnInit, OnDestroy {
    * Currently only supports "body".
    */
   @Input() public container: string;
+  /**
+   * Specifies if popover should be closed after clicking outside
+   */
+  @Input() public popoverCloseOnClickOutside: boolean;
```

Comment:
Okay, but wouldn't `outsideClick` be a more Angular2-way name? There is no `on` prefix in events like `click`, `dropdown` etc